client/src/pages/workspace/Workspace.tsx

```diff
@@ -1,16 +1,23 @@
-import React, {useState} from "react";
+import React, {useEffect, useState} from "react";
 import Board from "./board/board";
-import {BoardEntity} from "../../model/entity/BoardEntity";
+import {IBoardEntity} from "../../model/entity/IBoardEntity";
 import {List, ListItem} from "@mui/material";
-import AddBoard from "./board/AddBoard";
+import AddBoard from "./board/addBoard/AddBoard";
+import {getBoardsService} from "../../services/board";
 
 const Workspace: React.FC = () => {
-    const [boards, setBoards] = useState<BoardEntity[]>([
-        {id: 1, name: 'white board', description: 'mega ultra super cool red board', createdAt: new Date(), updatedAt: new Date()},
-        {id: 2, name: 'board2', description: 'boardDesc2', createdAt: new Date(), updatedAt: new Date()}
-    ]);
+    const [boards, setBoards] = useState<IBoardEntity[]>([]);
 
-    const addBoard = (newBoard: BoardEntity) => {
+    useEffect(() => {
+        getBoards();
+    }, [])
+
+    const getBoards = () => {
+        const fetchedBoards: IBoardEntity[] = getBoardsService();
+        setBoards(fetchedBoards);
+    }
```

Comment:
you can remove type for `fetchedBoards`, because it's obviously we expect `IBoardEntity[]`